megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -2918,6 +2919,14 @@ public Optional<WeaponMounted> getDisplayedWeapon() {
         }
     }
 
+    public Optional<AmmoMounted> getDisplayedAmmo() {
+        AmmoMounted ammo = unitDisplay.wPan.getSelectedAmmo();
+        if (null == ammo) {
```

Comment:
I believe 
`return Optional.ofNullable(unitDisplay.wPan.getSelectedAmmo());`
can replace this method's whole code.

Reply:
I believe I should have used that myself :)

Reply:
Alrighty, I'll plug that in.  Thanks!